src/net/sourceforge/kolmafia/request/ApiRequest.java

```diff
@@ -396,7 +396,8 @@ public static final void parseStatus(final JSONObject JSON) {
           Map.entry(
               "neverendingparty", Map.entry("neverendingPartyAlways", "_neverendingPartyToday")),
           Map.entry("voterregistered", Map.entry("voteAlways", "_voteToday")),
-          Map.entry("boxingdaycare", Map.entry("daycareOpen", "_daycareToday")));
+          Map.entry("boxingdaycare", Map.entry("daycareOpen", "_daycareToday")),
+          Map.entry("hascosmicball", Map.entry("hasCosmicBowlingBall", "_null")));
```

Comment:
`Map.entry("hasCosmicBowlingBall", "_null")` indicates that `PREF_TO_COOL_ITEM` probably has the wrong type. I would use a class with properties `coolItem`, `alwaysPref` and `todayPref`, all Strings. It only is called using `forEach`, so a list of those objects works.

That should avoid the special casing `if (coolItem != "hascosmicball")`: you can check whether `todayPref` is null instead.